### PR TITLE
Feature/revenue graph component fro dashboard

### PR DESCRIPTION
**Pull request by IgorZ07**
Screen Shot 2019-09-11 at 11 42 07 AM

### CODE REVIEW

File: src/components/revenue-graph/RevenueGraph.vue [hunk 1]
```diff
@@ -0,0 +1,97 @@
+<template>
+  <div class="revenue-graph">
+    <div v-for="month in revenue" :key="month.month" class="wrapper">
+      <div class="bar" :style="{height:barHeight(month.revenue)+'%'}"></div>
```
Comment (KevinTss) [inline src/components/revenue-graph/RevenueGraph.vue]: It's better to use string interpolation 
```
{ height: `${barHeight(month.length)}%` }
```

File: src/components/revenue-graph/RevenueGraph.vue [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    currency: { type: String, default: "EUR" },
+    multiplication: { type: Number, default: 1 }
+  },
+  data() {
+    return {};
+  },
+  computed: {},
+  methods: {
+    currentMonth() {
+      let months = [
```
Comment (KevinTss) [inline src/components/revenue-graph/RevenueGraph.vue]: Here is better to user momentjs, see [here](https://momentjs.com/docs/#/displaying/)

File: src/components/revenue-graph/RevenueGraph.vue [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        "Oct",
+        "Nov",
+        "Dec"
+      ];
+      let now = new Date();
+      let thisMonth = months[now.getMonth()];
+      return thisMonth;
+    },
+    barHeight(monthly) {
+      console.log(this.revenue);
```
Comment (KevinTss) [inline src/components/revenue-graph/RevenueGraph.vue]: Remove console.log

File: stories/index.js [hunk 4]
```diff
@@ -225,3 +225,24 @@ storiesOf("Mobile Offer Header", module)
       userSurname="Travolta"
       />`
   );
+
```
Comment (KevinTss) [inline stories/index.js]: You can place it in a separate file

**Comment by IgorZ07** [comment]
With grey bars if no revenue = 0
Screen Shot 2019-09-11 at 12 17 09 PM

File: src/components/revenue-graph/RevenueGraph.vue (hunk 2 above)
Comment (IgorZ07) [reply src/components/revenue-graph/RevenueGraph.vue]: importing moment, I have error on reload 
Screen Shot 2019-09-13 at 2 13 41 PM

Comment (KevinTss) [reply src/components/revenue-graph/RevenueGraph.vue]: @IgorZ07 Can you push your actual code? I will pull it to see what's happened

Comment (IgorZ07) [reply src/components/revenue-graph/RevenueGraph.vue]: Pushed as requested

**Pull request merged**